Workflow automation/Disable-AAD-Account-Revoke-Tokens/README.md

```diff
@@ -0,0 +1,66 @@
+
+Open-source tools such as MicroBurst or PowerZure, developed for research objectives, are also used maliciously simply by weaponize them. <br>
+These tools allow a malicious actor to assess and exploit resources within Microsoft cloud platforms by leveraging a compromised Azure Active Directory account and/or its token. <br>
+Azure Resource Manager (ARM) is the deployment and management service for Azure. <br>
+It provides the management layer that enables you to create, update, and delete resources in your Azure account. <br>
+It can be leveraged either via Azure Portal, via Rest API or using PowerShell, Azure CLI and SDKs. Read more about Azure Resource Manager here https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/overview.
+
+This management layer is crucial; therefore, it is important to protect it. <br>
+Microsoft Defender for Resource Manager protects against potential attacks including the use of exploitation tools like MicroBurst or PowerZure which will leverage compromised account and their tokens to authenticate and exploit the environment for privilege escalation, lateral movement, persistence, and more. Read more about the Microsoft Defender for Resource Manager here https://docs.microsoft.com/en-us/azure/defender-for-cloud/defender-for-resource-manager-introduction. <br>
+The authentication bearer token is an access token that contains claims that you can use in Azure Active Directory to identify the granted permissions to your API. <br>
+Once an attack is detected by Defender for Resource Manager, if an Azure Active Directory (Azure AD) Account has been utilized, you will need to act promptly and mitigate the compromised account. <br>
+Of course, you can do it manually, but automated response will ensure that the proper mitigation is indeed applied. <br>
+If an account is compromised you would disable the account temporarily, revoke all the associated authentication token, and reset the password. To automate this process, you can use this Azure Logic App we have developed to disable the account, revoke all the active tokens and notify the account’s manager if it exists or simply to a designated email address.
+You can deploy the Azure Logic App in your Subscription and use it with the Defender for Cloud Workflow Automation configured for Alerts generated from the Defender for Resource Manager.
+
+
+Note: This Log App will not be able to disable users if they are eligible or have active admin roles.<br>
+
+
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FGastori%2FLogicApps%2Fmain%2Fmdc-disable-account-revoke-token%2Fazuredeploy.json" target="_blank">
+    <img src="https://aka.ms/deploytoazurebutton"/>
+</a>
+<a href="https://portal.azure.us/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FGastori%2FLogicApps%2Fmain%2Fmdc-disable-account-revoke-token%2Fazuredeploy.gov.json" target="_blank">

```

Comment:
Please make sure to update the deployment link to point to the Defender for Cloud repo.